odkx-src/index.rst

```diff
@@ -7,7 +7,7 @@ ODK-X documentation
 
 .. _odk-2-introduction:
 
-The ODK-X Tool Suite is free and open-source software for collecting, managing, and using data in resource-constrained environments.
+The ODK-X Tool Suite is free and open-source software for collecting, managing and using data in resource-constrained environments.
```

Comment:
Per the style guide, the comma should be here, see: https://docs.odk-x.org/docs-style-guide/#serial-comma (I know different countries/styles do or don't use the penultimate/Oxford comma so it's a confusing one).

Reply:
Ok i see